streams/src/main/java/org/apache/kafka/streams/kstream/internals/KGroupedStreamImpl.java

```diff
@@ -235,7 +223,7 @@ private void determineIsQueryable(final String queryableStoreName) {
                                                                   final Aggregator<? super K, ? super V, T> aggregator,
                                                                   final Windows<W> windows,
                                                                   final Serde<T> aggValueSerde) {
-        return windowedBy(windows).aggregate(initializer, aggregator, aggValueSerde);
+        return windowedBy(windows).aggregate(initializer, aggregator);
```

Comment:
Should we really ignore the user passed `aggValueSerde`, even if it is a deprecated function? I think to achieve correctness we should still wrap it in `Materialized`?